template/.eslintrc.js

```diff
@@ -0,0 +1,35 @@
+module.exports = {
+    root: true,
+    // extends: ['plugin:echobind/react-native'],
+    extends: [
+      '@react-native-community',
+      'plugin:import/errors',
+      'plugin:import/warnings',
+      'plugin:import/typescript',
+    ],
+    rules: {
+      // enable i18n first
```

Comment:
this brings up another talking point. Should we include i18n out of the box? I feel like we should since it 1) makes testing less brittle and 2) allows for multi-language should an app ever need it.

The disadvantage is that it's slightly more work to add translation keys.

Reply:
I think we need to spike out i18n for a react-native app. Its probably quite similar but Im not 100% certain its implementation is 1:1 with a web-based app.

This library looks promising: https://github.com/react-native-community/react-native-localize

Reply:
I agree Ryan, that is what I was looking at, but def spike worthy! I'll add an issue to track it

Reply:
perfect thanks. `react-native-localize` is exactly what I was wanting to use, though I believe you need to combine it with something like [ii18n-js](https://github.com/fnando/i18n-js).

@rynatk totally agree its not 1:1 with web but it should be similar conceptually.